tests/test_helper/bats/central_control.bats

```diff
@@ -0,0 +1,61 @@
+load "${ABS_TOP_TEST_DIRNAME}test_helper/setup_teardown/$(basename "${BATS_TEST_FILENAME//.bats/.bash}")"
+
+setup() {
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/common.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/lxd.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/microovn.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-support/load.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-assert/load.bash
+
+    # Ensure TEST_CONTAINERS is populated, otherwise the tests below will
+    # provide false positive results.
+    assert [ -n "$TEST_CONTAINERS" ]
+}
+
+central_register_test_functions() {
+    bats_test_function \
+        --description "Testing of central node migration" \
+        -- central_tests
+    bats_test_function \
+        --description "Testing of database functionality" \
+        -- central_db_test
+}
+
+central_tests() {
```

Comment:
Couple notes regarding the tests:
* I think that using hardcoded container names is bit fragile. You can turn the `$TEST_CONTAINERS` into an array ([example](https://github.com/canonical/microovn/blob/09213bb29ce2da8d0ae5c2d6e6163d3f56f03996/tests/test_helper/bats/ovsdb_schema_upgrade.bats#L30)) and access names via indexes
* I'd prefer checks after every step, rather than at the end
* In addition to checking microovn status and snap service status, we could also check ovsdb cluster status via `ovn-appctl`. There's already [a function that ensures that server left the cluster](https://github.com/canonical/microovn/blob/09213bb29ce2da8d0ae5c2d6e6163d3f56f03996/tests/test_helper/microovn.bash#L458-L467) but similar will probably have to be made to ensure that server joined.